.github/workflows/python-perf.yml

```diff
@@ -0,0 +1,113 @@
+name: Python Performance
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+    paths:
+      - '.github/workflows/python-perf.yml'
+      - 'binding/python/__init__.py'
+      - 'binding/python/_koala.py'
+      - 'binding/python/_util.py'
+      - 'binding/python/test_koala_perf.py'
+      - 'lib/common/**'
+      - 'lib/linux/**'
+      - 'lib/jetson/**'
+      - 'lib/mac/**'
+      - 'lib/raspberry-pi/**'
+      - 'lib/windows/**'
```

Comment:
resources/audio_samples should also trigger this action right?

Reply:
I thought about it and I don't think it's necessary, but there's also no harm in adding it here